### ISSUE TITLE
Unable to override username even when protection is disabled.

### ISSUE THREAD

**Issue by slopes1967** [jaszhix/NoMansConnect#104]
- Description of the problem

One of two issues actually.

I have two systems I play on. One, NMC found the right username on my steam (slopes1967), but when I went to install NMC on the other system, it used old steam account name that my son uses (Portuguese Fireman). So, I went into the setup and tried to override the name back to the one I play on, and it tells me that protection must be disabled. It's not enabled according to the settings, so i'm not sure what's up.

So first, I'm wondering why I can't override the username when it's not protected,  and second, why it used my other steam account as the default, when my other system didn't. (I own 2 copies of NMS. Weird how it decided to use that one.

- Windows version    Windows 10 latest update
- NMC version           NMC 1.5.4
- Absolute path to your storage_x.hg files. 

C:\Users\steve\AppData\Roaming\HelloGames\NMS\st_76561198120735526

- Absolute path to your NMS install directory.

F:\Program Files (x86)\Steam\steamapps\common\No Man's Sky

Use Pastebin of directly attach these files (Users/<Username>/AppData/Roaming/NoMansConnect):

- NMC.log             https://pastebin.com/6RPUyqiD
- settings.json       https://pastebin.com/9gTSzV7Y

**Comment by jaszhix** [comment]
It will use whatever username is in the save file. If there are multiple save file directories in `C:\Users\steve\AppData\Roaming\HelloGames\NMS` then you might need to specify the exact directory in the app settings. You don't mention which username its saying is protected - there is no "Portuguese Fireman" user on the server.

**Comment by slopes1967** [comment]
So, I ended up copying the save file from my other machine that loads NMC with the right profile and replaced it with the save file on the machine I'm having issues with. I also made sure the path was correct and I had no other save files on that system. It still loaded the Portuguese Fireman username and wouldn't let me override it.. If the user name is stored in the save file, I thought for sure copying the one from my other system might do the trick.. So much for that! :) It's grabbing it from someplace, I just have to keep looking. I'm still perplexed why I can't override it, when protection isn't enabled..
